alacritty_macos.yml

```diff
@@ -43,12 +43,18 @@ font:
 # Should display the render timer
 render_timer: false
 
+# Should invert the cursor color. If false, display the cursor in the
+# cursorforeground and cursorbackground colors
+invert_cursor: true
```

Comment:
I think I'd rather see this option called `custom_cursor_colors`--this makes it opt-in instead of opt-out. The name will need to be updated throughout the patch.